notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>remove code cell, this step is already done above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>I use this to reset the timestamp for a new run, but the job has a different name so I'll remove it</p>